### DISCUSSION TITLE
firefox Reader View mode support

### DISCUSSION THREAD

**Discussion by chizhang** [philc/vimium#4529] [General]
Is it possible the Vimium to support the Firefox Reader View mode to navigate articles by using the keyboard? The Vimium seems is disabled in this mode.

**Comment by Bestboigill** [comment]
i dont think it is possible you would want to download a different browser that is made for it like qt browser

**Comment by philg-dev** [comment]
I've taken a look at it for a bit just now - here's what I've found:

- Firefox Reader mode runs in a special context that has nothing to do with the original site's DOM anymore
- Extension scripts aren't even loaded at all in the reader view
- Discussion about [Content scripts in reading mode](https://discourse.mozilla.org/t/content-scripts-in-reading-mode/24944)
- [Bugzilla Ticket on the topic](https://bugzilla.mozilla.org/show_bug.cgi?id=1371786)

So currently it looks like it's actually impossible to achieve this feature with any addon.
